core/pythonAction/pythonaction.py

```diff
@@ -61,7 +61,7 @@ def run():
     result = None
     try:
         exec(flask.g, namespace)
```

Comment:
why isn't this part of init: `exec(flask.g, namespace)` - wouldn't that save the eval overhead on every run?